proposals/0216-widget-support.md

```diff
@@ -522,62 +587,157 @@ This functional group should be provided to the app developers as a conditional
+
+      This parameter is related to widgets only. The default main window, which is pre-created during app registration, will be created based on the HMI types specified in the app registration request.
+    </descripion>
+  </param>
+</function>
+```
+
+Allows an app to create a widget related to a specific HMI type.
+As an example if a `MEDIA` app becomes active, this app becomes audible and is allowed to play audio. Actions such as skip or play/pause will be
+directed to this active media app. In case of widgets, the system can provide a single "media" widget will acts as a placeholder for the active media app.
+
+It is only allowed to have one window per HMI type. This means that a media app can only have a a single MEDIA widget. Still the app can create widgets omitting this parameter or with specifying the HMI type `DEFAULT`. Those widgets would be available to the user independent of the HMI type.
+
+This parameter is related to widgets only. The default main window, which is pre-created during app registration, will be created based on the HMI types specified in the app registration request.
+
+The benefit of `AppHMIType` is the linkage to embedded features such as embedded navigation and media/radio. Another benefit is the the ability to use policy management. The application can only create widgets for app types that are allowed according to policies.
+
+### Service type
+
+```xml
+<function name="CreateWindow" messagetype="request">
+  :
+  <param name="widgetType" type="AppServiceType" mandatory="false">
+    <description>
+      Allows an app to create a widget related to a specific service type.
+      As an example if a `MEDIA` app becomes active, this app becomes audible and is allowed to play audio. Actions such as skip or play/pause will be
+      directed to this active media app. In case of widgets, the system can provide a single "media" widget will acts as a placeholder for the active media app.
+      
+      It is only allowed to have one window per service type. This means that a media app can only have a single MEDIA widget. Still the app can create widgets omitting this parameter. Those widgets would be available as app specific widgets that are permanently included in the HMI.
+
+      This parameter is related to widgets only. The default main window, which is pre-created during app registration, will be created based on the HMI types specified in the app registration request.
+    </descripion>
+  </param>
+</function>
+```
+
+Allows an app to create a widget related to a specific service type.
+As an example if a `MEDIA` app becomes active, this app becomes audible and is allowed to play audio. Actions such as skip or play/pause will be
+directed to this active media app. In case of widgets, the system can provide a single "media" widget will acts as a placeholder for the active media app.
```

Comment:
```suggestion
directed to this active media app. In case of widgets, the system can provide a single "media" widget which will act as a placeholder for the active media app.
```